core/badge-urls/make-badge-url.js

```diff
@@ -71,9 +71,38 @@ function staticBadgeUrl({
   return `${baseUrl}/badge/${path}.${format}${suffix}`
 }
 
+function queryStringStaticBadgeUrl({
+  baseUrl = '',
+  label,
+  message,
+  color = 'lightgray',
```

Comment:
How about defaulting to no color, since that's allowed in the query string badge? In the path-based static badge I think the color needs to be included regardless.

Reply:
:+1: